cmd/main.go

```diff
@@ -79,7 +78,9 @@ func init() {
 
 	rootCmd.Flags().AddGoFlagSet(flag.CommandLine)
 	rootCmd.Flags().StringVar(&endPoint, "endpoint", "unix://tmp/csi.sock", "CSI endpoint")
-	rootCmd.Flags().StringVar(&driverName, "drivername", string(v1.SharedResourcesCSIDriver), "name of the driver")
+	//rootCmd.Flags().StringVar(&driverName, "drivername", string(v1.SharedResourcesCSIDriver), "name of the driver")
```

Comment:
Do we still need this commented out line?

Reply:
I'd like to leave it as per the TODO comment below we'll swtich back to it once you've updated that driver name with your openshift/api PR